lnst/Controller/Machine.py

```diff
@@ -279,6 +279,13 @@ def get_dev_by_hwaddr(self, hwaddr):
                 return dev
         return None
 
+    def get_dev_by_ifname(self, ifname, netns=None):
+        ns_instance = self._get_netns_by_name(netns)
+        for dev in list(self._device_database[ns_instance].values()):
```

Comment:
yeah, this is a construct that is typically auto created by the `2to3` scripts which we used a while ago, we have it in a couple of places since the migration, since it's harmless i didn't really care about removing them.

but for new code i think it's unnecessary.

Reply:
fixed